packages/web-twig/src/Resources/components/TextField/stories/TextFieldValidation.twig

```diff
@@ -1,56 +1,37 @@
-<TextField
-    id="textfieldDanger"
-    isRequired
-    label="Label"
-    name="textfieldDanger"
-    placeholder="Placeholder"
-    validationState="danger"
-    value="Filled"
-/>
-<TextField
-    id="textfieldDangerMessage"
-    isRequired
-    label="Label"
-    message="Validation message"
-    name="textfieldDangerMessage"
-    placeholder="Placeholder"
-    validationState="danger"
-    value="Filled"
-/>
 <TextField
     id="textfieldSuccess"
-    isRequired
     label="Label"
-    message="Validation message"
     name="textfieldSuccess"
     placeholder="Placeholder"
     validationState="success"
     value="Filled"
 />
 <TextField
     id="textfieldWarning"
-    isRequired
     label="Label"
-    message="Validation message"
+    validationText="Validation text"
     name="textfieldWarning"
     placeholder="Placeholder"
     validationState="warning"
     value="Filled"
 />
 <TextField
-    id="textfieldDisabled"
-    isDisabled
+    id="textfieldDanger"
     label="Label"
-    message="Message"
-    name="textfieldDisabled"
+    validationText="{{ ['First validation text', 'Second validation text'] }}"
```

Comment:
The messages are slightly different in the web package:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/50e3abd9-a614-479d-80e9-8ec4e9c46f7a)